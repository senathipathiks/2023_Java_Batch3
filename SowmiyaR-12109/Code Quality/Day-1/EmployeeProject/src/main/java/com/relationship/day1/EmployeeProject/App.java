package com.relationship.day1.EmployeeProject;

/**
 * Hello world!
 *
 */
public class App 
{
    public static void main( String[] args )
    {
        System.out.println( "Hello World!" );
    }
}
