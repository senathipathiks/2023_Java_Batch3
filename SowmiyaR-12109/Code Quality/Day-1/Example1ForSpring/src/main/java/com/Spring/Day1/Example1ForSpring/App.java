package com.Spring.Day1.Example1ForSpring;

import org.springframework.beans.factory.BeanFactory;
import org.springframework.context.ApplicationContext;
import org.springframework.context.support.ClassPathXmlApplicationContext;

/**
 * Hello world!
 *
 */
public class App 
{
    public static void main(String[] args )
    {
        ApplicationContext context=new ClassPathXmlApplicationContext("Config.xml");
//        BeanFactory context=new ClassPathXmlApplicationContext("Config.xml");
          Student obj1=(Student)context.getBean("s1");
          System.out.println(obj1);
       
          Student obj2=(Student)context.getBean("s2");
          System.out.println(obj2);
    }
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
}


