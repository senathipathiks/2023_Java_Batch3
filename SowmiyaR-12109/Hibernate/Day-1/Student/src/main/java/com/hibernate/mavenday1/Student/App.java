package com.hibernate.mavenday1.Student;

import org.hibernate.Session;
import org.hibernate.SessionFactory;
import org.hibernate.Transaction;
import org.hibernate.cfg.Configuration;

/**
 * Hello world!
 *
 */
public class App 
{
    public static void main( String[] args )
    {
//        Student s1=new Student(1,"Sowmi","chennai");
//        Student s2=new Student(2,"Hari","Mumbai");
//    	Student s3=new Student(4,"Sherin","Trichy");
//        
//        Configuration cfg = new Configuration().configure("hibernate.cfg.xml");
//        SessionFactory factory=cfg.buildSessionFactory();
//        Session session=factory.openSession();
//        Transaction t=session.beginTransaction();
////        session.save(s1);
//        session.save(s3);
//        t.commit();
//    	================================================================
    	
    	Employee e1=new Employee(1,"sowmi","chennai");
    	Configuration cfg = new Configuration().configure("hibernate.cfg.xml");
      SessionFactory factory=cfg.buildSessionFactory();
      Session session=factory.openSession();
      Transaction t=session.beginTransaction();
//      session.save(s1);
      session.save(e1);
      t.commit();
    	
    
    }
}
