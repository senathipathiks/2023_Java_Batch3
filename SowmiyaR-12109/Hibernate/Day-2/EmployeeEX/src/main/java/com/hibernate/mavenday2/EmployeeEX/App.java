package com.hibernate.mavenday2.EmployeeEX;

import java.util.List;
import java.util.Scanner;

import org.hibernate.Session;
import org.hibernate.SessionFactory;
import org.hibernate.Transaction;
import org.hibernate.cfg.Configuration;


/**
 * Hello world!
 *
 */
public class App 
{
    public static void main( String[] args )
    {
    	
    	Configuration cfg = new Configuration().configure("hibernate.cfg.xml");
      SessionFactory factory=cfg.buildSessionFactory();
      Session session=factory.openSession();
      Transaction t=session.beginTransaction();
	  String msg=null;

//      ===========================Insert
//      Employee e1=new Employee();
//      
//      e1.setName("Magesh");
//      session.save(e1);
//      t.commit();
      
//      ==========================Update
//      Employee temp=session.find(Employee.class, 3);
//      temp.setName("Hari");
//      session.update(temp);
//         t.commit();

//      =====================Delete
//      Employee e2=session.find(Employee.class, 5);
//      session.delete(e2);
//      t.commit();
//      =============Shows table list
//      List<Employee> empList=session.createQuery("from Employee").list();
//      for(Employee e:empList)
//      {
//    	System.out.println(e);  
//      }
//      System.out.println("Done");
      
//      ===========================Switch
  	do {

    	  System.out.println("1.insert\n2.delete\n3.update\n4.select \n5.exit");
    	  Scanner sc= new Scanner(System.in);
    	  System.out.println("Enter your choice");
    	  int k=sc.nextInt();
    	
			
    	  switch(k) {
    	  case 1:
    		  System.out.println("Enter name: ");
    		  Employee e1=new Employee();
    		  String name=sc.next();
              e1.setName(name);
    	      session.save(e1);
    	      System.out.println("Inserted successfully"); 
    	     t.commit(); 
    	      break;
    	       
    	      
    	  case 2:
    		  System.out.println("Enter id:");
    		  int id1=sc.nextInt();
    		  Employee e2=session.find(Employee.class, id1);
    	      session.delete(e2);
    	      System.out.println("deleted successfully");
    	      t.commit();
    	      break;
    	  case 3:
    		  System.out.println("Enter id:");
    		  int id2=sc.nextInt();
    		  String name1=sc.next();
    		  Employee temp=session.find(Employee.class, id2);
    	       temp.setName(name1);
    	      session.update(temp);
    	      System.out.println("Updated successfully");
    	         t.commit();
    	         break;
    	  case 4:
    		  
    		  List<Employee> empList=session.createQuery("from Employee").list();
    	      for(Employee e:empList)
    	      {
    	    	System.out.println(e);  
    	      }
    	      System.out.println("Done");
    	      break;
    	  case 5:
    		  System.out.println("Exit");
    		  break;
    	  }
    	  System.out.println("do you want to continue[yes/no]");
    	  msg=sc.next();
		}while (msg.equals("yes"));
  	    System.out.println("Terminated");
        
    	 
    	  
      }
      
    }

