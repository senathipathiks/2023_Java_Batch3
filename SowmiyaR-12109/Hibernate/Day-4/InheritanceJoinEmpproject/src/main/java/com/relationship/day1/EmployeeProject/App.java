package com.relationship.day1.EmployeeProject;

import java.util.ArrayList;

import org.hibernate.Session;
import org.hibernate.SessionFactory;
import org.hibernate.Transaction;
import org.hibernate.cfg.Configuration;

/**
 * Hello world!
 *
 */
public class App 
{
	public static void main(String[] args) {
	Configuration cfg=new Configuration().configure("hibernate.cfg.xml");
    SessionFactory factory=cfg.buildSessionFactory();
    Session session=factory.openSession();
     Transaction t=session.beginTransaction();
     
      
//     Department d1=new Department(001,"sales");
//     Department d2=new Department(002,"marketing");
//     
//    Employee e1=new Employee(005,"Charan",d1);
//    Employee e2=new Employee(006,"Sowmi",d1);
//    Employee e3=new Employee(007,"Hari",d2);
//    
//    session.persist(e1);
//    session.persist(e2);
//    session.persist(e3);
//    t.commit();
//    -----------------------------------------------------
    
     
     
     Employee1 e1=new RegularEmp(101,"Hari",10000.00f,1000);
     Employee1 e2=new ContractEmp(102,"Charan",1000.00f,"2 years");
     session.save(e1);
     session.save(e2);
     t.commit();
     System.out.println("Success");
}
}
