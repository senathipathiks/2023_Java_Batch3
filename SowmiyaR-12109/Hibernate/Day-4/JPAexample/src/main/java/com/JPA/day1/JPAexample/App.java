package com.JPA.day1.JPAexample;

import javax.persistence.EntityManager;
import javax.persistence.Persistence;

/**
 * Hello world!
 *
 */
public class App 
{
    public static void main( String[] args )
    {
        EntityManager emf=Persistence.createEntityManagerFactory("CourseDB").createEntityManager();
        emf.getTransaction().begin();
        Course c1=new Course(103,"C++","EEE");
        emf.persist(c1);
        emf.getTransaction().commit();
        emf.close();
    }
}
