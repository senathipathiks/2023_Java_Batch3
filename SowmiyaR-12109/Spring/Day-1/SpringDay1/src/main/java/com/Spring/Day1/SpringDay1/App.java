package com.Spring.Day1.SpringDay1;

import org.springframework.context.ApplicationContext;
import org.springframework.context.support.ClassPathXmlApplicationContext;

/**
 * Hello world!
 *
 */

public class App 
{
    public static void main( String[] args )
    {
    	
//        Vehicle obj=new Car();
//        obj.run();
//        Vehicle obj1=new Bike();
//        obj1.run();
//    ===================================
    	
    	ApplicationContext context=new ClassPathXmlApplicationContext("Config.xml");
    	Vehicle v1=(Vehicle)context.getBean("b1");
    	v1.run();
    	Vehicle v2=(Vehicle)context.getBean("c1");
    	v2.run();
    }
}
