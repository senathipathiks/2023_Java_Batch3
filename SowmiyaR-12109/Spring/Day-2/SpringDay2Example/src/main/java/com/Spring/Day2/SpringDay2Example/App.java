package com.Spring.Day2.SpringDay2Example;

import org.springframework.context.ApplicationContext;
import org.springframework.context.support.ClassPathXmlApplicationContext;

/**
 * Hello world!
 *
 */
public class App 
{
    public static void main( String[] args )
    {
    	ApplicationContext context=new ClassPathXmlApplicationContext("Config.xml");
    	Student s1=(Student)context.getBean("student");
    	s1.setScity("mumbai");
    	System.out.println(s1);
    	Student s2=(Student)context.getBean("student");
    	System.out.println(s2);
    	
    }
}
