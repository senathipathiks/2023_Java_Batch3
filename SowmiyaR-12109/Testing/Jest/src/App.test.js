// import { render, screen } from '@testing-library/react';
import App from './App';
import renderer from 'react-test-renderer';

describe('Test suite for App',()=>{

  test("Snapshot test",()=>{
    const comp=renderer.create(<App/>)
    const tree=comp.toJSON();
    expect(tree).toMatchSnapshot();
  })




  // test('renders learn react link', () => {
  //   render(<App />);
  //   const linkElement = screen.getByText("Welcome to my page!!");
  //   expect(linkElement).toBeInTheDocument();
  // });

  // it('renders Welcome to relevantz', () => {
  //   render(<App />);
  //   const linkElement = screen.getByText("Hi Welcome to relevantz");
  //   expect(linkElement).toBeInTheDocument();
  // });

  // it('renders Welcome to jest', () => {
  //   render(<App />);
  //   const linkElement = screen.getByText("Hello Welcome to jest");
  //   expect(linkElement).toBeInTheDocument();
  // });
});


