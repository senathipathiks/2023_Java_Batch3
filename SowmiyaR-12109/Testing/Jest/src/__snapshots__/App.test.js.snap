// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suite for App Snapshot test 1`] = `
<div
  className="App"
>
  <h1>
    Welcome to my page!!
  </h1>
  <br />
  <h2>
    Hi Welcome to relevantz
  </h2>
  <br />
  <h2>
    Hello Welcome to jest
  </h2>
  <br />
  <h2>
    Updated Jest
  </h2>
</div>
`;
